samwise.js

```diff
@@ -9,15 +9,14 @@
 ( function( document ) {
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?